external/deep-object-reid/configs/ote_custom_classification/mobilenet_v3_large_1/main_model_multilabel.yaml

```diff
@@ -11,7 +11,7 @@ lr_finder:
   n_trials: 15
 
 model:
-  name: 'mobilenetv3_large_21k'
+  name: 'mobilenetv3_large'
```

Comment:
Is there any reason why we have to revise the deep-object-reid side?

Reply:
Current implementation builds the model using timms_model_wrapper, which is not supported in MPA. (All the state dict keys can't be matched with MPA model). 
If we replace current supervised recipe by our MPA implementation, this modification is not needed. But this PR does not include supervised recipe replacement so I think this modification is reasonable in perspective of stability.

Reply:
Yes, it works fine